blocks/cards/cards.css

```diff
@@ -1,3 +1,12 @@
+:where(.cards-card-body) {
```

Comment:
Is there a difference between this and simply`.cards-card-body`?

Reply:
There is a difference of specificity but we don't need that in this case. Updated.